rust/src/transformations/make_stable_expr/namespace_dt/expr_datetime_component/test.rs

```diff
@@ -0,0 +1,33 @@
+use crate::metrics::SymmetricDistance;
+use crate::transformations::make_stable_expr::namespace_dt::test::{
+    example_dt_lf, parse_example_dt_lf,
+};
+use crate::transformations::make_stable_lazyframe;
+
+use super::*;
+
+#[test]
+fn test_make_expr_components() -> Fallible<()> {
+    let (lf_domain, lf) = example_dt_lf()?;
```

Comment:
Slight preference to keep the example data in the same file, even at the cost of some redundancy. I want to be able to look at the test and see that it's doing the right thing, rather than making opaque assertions.